### PR TITLE
Prevent the get trap for the proxy from taking effect if called with internal Symbols or internal 'inspect" prop key

### LINKED ISSUE #1: cant console log a type
Get the following error:

`TypeError: Cannot convert a Symbol value to a string`

### PR DESCRIPTION
**Pull request by alexkolson**
turns out when you call `console.log(proxiedThing)` node calls the get trap with built in Symbols
In the `throw TypeError` statement we subsequently have issues converting the Symbol to a string value implicitly.
I opted to just check if the propertty is a symbol or if its the inspect internal string and if it is just short circuit the get trap.

See also: https://github.com/nodejs/node/issues/10731

### CODE REVIEW

File: src/create-type.test.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { canonize, createType } from './create-type'
 
 describe('KISS type', () => {
   describe('in its most basic form', () => {
-    const type = createType(['prop'])
+    const type = createType(['prop', 'baz', 'bar'])
```
Comment (alexbepple) [inline src/create-type.test.js]: Why do we need more props?

File: src/create-type.test.js [hunk 2]
```diff
@@ -22,6 +22,14 @@ describe('KISS type', () => {
...
       __.assertThat(() => type.set.unknown, failsNicely)
     })
+    it('type properties can be console logged', () => {
+      const doesNotThrowError = __.not(
+        __.throws(
+          __.typedError(TypeError, __.containsString('Cannot convert a Symbol value to a string'))
+        )
+      )
+      __.assertThat(() => console.log(type.props), doesNotThrowError)
+    })
```
Comment (alexbepple) [inline src/create-type.test.js]: What do you think about just `console.log(type.props)` as the assertion (in the sense that there are not runtime errors)?

**Comment by alexbepple** [comment]
What is the reason for the changes to `yarn.lock`?
What is the reason for seemingly unrelated changes, e.g. newline at eof in `package.json`?
If these changes are truly unrelated, let's get rid of them.

Currently, `yarn lint` fails. (Yes, I know, CI is not set up. Sorry.) 

Do you want to work on the PR or shall I just make the changes myself?

File: src/create-type.test.js (hunk 1 above)
Comment (alexkolson) [reply src/create-type.test.js]: we dont. Good point.

File: src/create-type.test.js (hunk 2 above)
Comment (alexkolson) [reply src/create-type.test.js]: im a fan

**Comment by alexkolson** [comment]
@alexbepple done

**Pull request merged**